emptytheme/experimental-theme.json

```diff
@@ -0,0 +1,37 @@
+{
+	"global": {
+		"settings": {
+			"color": {
+				"gradients": [ ],
+				"palette": [ ]
+			},
+			"typography": {
+				"customFontSize": true,
+				"customLineHeight": true,
+				"fontFamilies": [ ],
+				"fontSizes": [ ],
+				"fontStyles": [ ],
```

Comment:
I guess in most cases one should not have to define fontStyles, fontWeights, textDecorations, and textTransforms. I guess the future one may define per font family. These API's are very new I would probably not include them yet as they will probably change.

Reply:
Yeah, that all makes sense. I've removed for now.